libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -510,7 +512,6 @@ class AP_InertialSensor : AP_AccelCal_Client
     // health of gyros and accels
     bool _gyro_healthy[INS_MAX_INSTANCES];
     bool _accel_healthy[INS_MAX_INSTANCES];
-
```

Comment:
Why removing this? It's a good blank line 🙂

Reply:
mistake